tests/karma/unit/controllers/inbox.js

```diff
@@ -160,7 +157,6 @@ describe('InboxCtrl controller', function() {
       chai.assert(spyState.go.called, 'Should change state');
       chai.expect(spyState.go.args[0][0]).to.equal(spyState.current.name);
       chai.expect(spyState.go.args[0][1]).to.deep.equal({ id: null });
-      chai.assert(snackbar.called, 'Should display toast');
```

Comment:
Ugh. Snackbar is still there and works perfectly. But because we can't mock $translate (don't get me started) the call never returns in unit testing.

I'm seriously wondering if we need to wrap `$translate` in our own translate service, just so we can mock it.